libraries/core-react/stories/components/Popover.stories.tsx

```diff
@@ -1,8 +1,11 @@
-import React from 'react'
+import React, { useRef, useState } from 'react'
 import styled from 'styled-components'
 import { Typography, Button, Popover, PopoverProps, Card } from '@components'
 import { Meta, Story } from '@storybook/react'
 
+const { Title, Content } = Popover
```

Comment:
We should avoid the use of object destructing and conform on writing the stories by using` Popover.Title` as in other stories. 👍  In this way it's easy to see what this sub component is in "View source" in the story.